hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Constants.java

```diff
@@ -142,6 +142,10 @@ private Constants() {
   public static final String ASSUMED_ROLE_CREDENTIALS_DEFAULT =
       SimpleAWSCredentialsProvider.NAME;
 
+  /**
+   * Absolute path to the web identity token file
```

Comment:
nit, add a . at the end of the sentence. javadoc versions like that.

also in docs, say "path in local/mounted filesystem" so it is clear it is not a cluster fs like HDFS